fov_processing_pipeline/wrappers.py

```diff
@@ -11,19 +11,19 @@
 from fov_processing_pipeline import plots
 
 
-def row2im(df_row):
-    # take a dataframe row and returns an image in CZYX format with channels in order of
-    # Brightfield, DNA, Membrane, Structure, seg_dna, seg_membrane, seg_structure
+def row2im(df_row, ch_order=['BF', 'DNA', 'Cell', 'Struct']):
```

Comment:
Might be worthwhile to start using method docstrings like this:

https://github.com/AllenCellModeling/cookiecutter-pypackage/blob/master/%7B%7Bcookiecutter.project_slug%7D%7D/%7B%7Bcookiecutter.project_slug%7D%7D/example.py